src/wdio/services/AxeService.js

```diff
@@ -0,0 +1,64 @@
+/* global browser, axe */
+import fs from 'fs';
+
+let axeCoreSrc;
+
+/**
+* Webdriver.io AxeService
+* provides the browser.axe() command.
+*/
+export default class AxeService {
+  // eslint-disable-next-line class-methods-use-this
+  before(config) {
+    const axeConfig = {
+      inject: true, // True if axeCore script should be injected on each test page.
+      ...(config.axe || {}),
+    };
+
+    browser.addCommand('axe', (options) => {
+      // Conditionally inject axe. This allows consumers to inject it themselves
+      // in the test examples which would slightly speed up test runs.
+      if (axeConfig.inject) {
+        if (!axeCoreSrc) {
+          axeCoreSrc = fs.readFileSync(require.resolve('axe-core'), 'utf8');
+          axeCoreSrc = axeCoreSrc.replace(/^\/\*.*\*\//, '');
+        }
+        if (browser.execute('return window.axe === undefined;')) {
```

Comment:
Should it make sense to check this if first before creating the axeCoreSrc if its unavailable?

Reply:
good idea